src/topp/MSGFPlusAdapter.cpp

```diff
@@ -194,6 +197,10 @@ class MSGFPlusAdapter :
     setValidStrings_("fixed_modifications", all_mods);
     registerStringList_("variable_modifications", "<mods>", vector<String>(), "Variable modifications, specified using UniMod (www.unimod.org) terms, e.g. 'Oxidation (M)'", false);
     setValidStrings_("variable_modifications", all_mods);
+    
+    registerStringOption_("legacy_conversion", "<choice>", "false", "MS-GF+ search results are available as mzid or tsv. With legacy conversion switched on, the tsv are being used instead of the mzid", false, true);
```

Comment:
Yes, definitely! Also, this might be a better description: "Use the indirect conversion of MS-GF+ results to idXML via export to TSV. Try this only if the default conversion takes too long or uses too much memory."